BAU: Update PR template

kerrr opened the pull request:
<!-- Provide a general summary of your changes in the Title above -->
<!-- Include the Jira ticket number in square brackets as prefix, eg `[P4-XXXX] PR Title` -->

## Proposed changes

### What changed

Update the PR template with the base url for Jira tickets.

### Why did it change

To make it easier for us to populate the link.

merlinc commented on di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/domain/Attribute.java:
```diff
@@ -0,0 +1,25 @@
+
+public enum Attribute {
+
+    GIVEN_NAMES("givenNames"),
```
This is a smaller subset of what's in `di-ipv-core-back` - gotcha

Do you think we'll need a master list of these Enums somewhere at some point?

martin-bucinskas replied:
It is a smaller list for the time being,  as the address is being combined into a single JSON field in the /userinfo data. But ideally, we should be able to expose all via some common library or framework. 

The current attributes in the orchestrator is just for demo

kerrr committed:
BAU: Update PR template

Update the PR template with the base url for Jira tickets.

This makes it easier for us to populate the link.
